config/management/src/lib.rs

```diff
@@ -0,0 +1,212 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+#![forbid(unsafe_code)]
+
+use libra_secure_storage::{Storage, VaultStorage};
+use libra_types::waypoint::Waypoint;
+use std::{fmt::Write, str::FromStr};
+use structopt::StructOpt;
+
+pub mod constants {
+    pub const CONSENSUS_KEY: &str = "consensus";
```

Comment:
Question: do we want this to be the source of truth for all of these constant names? e.g., we currently use "consensus_key" in the key manager?

Reply:
You ask the hard questions, so I will ask you a harder question... We /need/ a unified constants "crate", where should that live? I mean honestly should this binary (management) be in config or secure? I am deeply torn by this hard decisions.... Let me know your thoughts.

Reply:
By the way, these are the values we should unify against regardless of whatever silliness is currently in the code...